app/models/organization.js

```diff
@@ -20,18 +20,18 @@ export default DS.Model.extend({
   plan: DS.attr('string'),
   opsAlertEmail: DS.attr('string'),
   securityAlertEmail: DS.attr('string'),
-  stripeSubscriptionId: DS.attr('string'),
-  stripeCustomerId: DS.attr('string'),
+  //stripeSubscriptionId: DS.attr('string'),
+  //stripeCustomerId: DS.attr('string'),
   users: DS.hasMany('user', {async:true}),
   invitations: DS.hasMany('invitation', {async:true}),
   roles: DS.hasMany('role', {async:true}),
   securityOfficer: DS.belongsTo('user', {async:true}),
   billingContact: DS.belongsTo('user', {async:true}),
-  billingDetail: DS.belongsTo('billing-detail', {async:true}),
-  allowPHI: Ember.computed.match('plan', /production|platform/),
-  hasStripeSubscription: Ember.computed.bool('stripeSubscriptionId'),
-  hasStripeCustomer: Ember.computed.bool('stripeCustomerId'),
-  hasStripe: Ember.computed.and('hasStripeCustomer', 'hasStripeSubscription'),
+  billingDetail: DS.belongsTo('billingDetail', {async:true}),
```

Comment:
All types passed to `belongsTo` and `hasMany` should be `dash-erized`, not `camelCase`.